src/legacy/ui/public/url/url.js

```diff
@@ -108,19 +140,7 @@ export function KbnUrlProvider($injector, $location, $rootScope, $parse, Private
       // remove filters
       const key = expr.split('|')[0].trim();
 
-      // verify that the expression can be evaluated
-      const p = $parse(key)(paramObj);
```

Comment:
i couldn't find a case where this is actually used, i always only saw expressions like `{{id}}`